apps/src/templates/RegionalPartnerSearch.jsx

```diff
@@ -38,7 +38,8 @@ const styles = {
 
 class RegionalPartnerSearch extends Component {
   static propTypes = {
-    responsiveSize: PropTypes.oneOf(['lg', 'md', 'sm', 'xs']).isRequired
+    responsiveSize: PropTypes.oneOf(['lg', 'md', 'sm', 'xs']).isRequired,
+    page: PropTypes.string
```

Comment:
I'd add a small comment here. It was not clear to me at all (until I scrolled down to see how it was used) what `page` means in this context.

Reply:
Good feedback.  I'll rename it to something like `sourcePageId`.